### ISSUE TITLE
Request: Add item spawner support

### ISSUE THREAD

**Issue by UnioDex** [timbru31/SilkSpawners#55]
Hi. Can you add item spawner support? I have iron spawner in my server for this reason i can not use SilkSpawners. I have a custom plugin but it is not that good like SilkSpawners.

**Comment by timbru31** [comment]
You mean iron ingots as the "mob" of the spawner?

**Comment by UnioDex** [comment]
Yes. Iron ingots, diamonds and something like that. You can put every item in a item spawner.

**Comment by timbru31** [comment]
What plugin offers these functionality? Never heard of this before, since the items are no mobs technically.

**Comment by UnioDex** [comment]
This offers: https://dev.bukkit.org/projects/item-spawner
But this is not even needed. You can take it with /setblock command.
/setblock ~ ~1 ~ minecraft:mob_spawner 0 replace {SpawnData:{id:"minecraft:item",Item:{id:iron_block,Count:1b}}}

**Comment by timbru31** [comment]
Okay, so basically it's just custom NBT data. It's already planned to let users define custom NBT data that is then copied to the item/block when obtained via /ss give or mined. This would also include custom item spawners like your example.

No ETA, but I hope to have more spare time in the next weeks.
